eve-api-data/src/main/java/com/eveonline/api/data/Contracts.java

```diff
@@ -6,7 +6,7 @@
 import java.util.Date;
 
 @Entity
-@Table(name = "apicontracts", schema = "eve")
```

Comment:
All removals like this are just what I consider to be a best practice.  The schema/database or both should be referenced as environment configurations.  The connection string, the datasources default db/schema... the user of the datasources default schema, etc.  Schema means different things to different db vendors :(.  MSSQL has database and schema... oracle just has schemas, mysql just has databases... db2 I believe only has schema's just like oracle, etc.